src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -730,7 +726,7 @@ void Convolution::initSupportedPrimitiveDescriptors() {
         if (containJitImpl && isPossibleToSkipInitConfig(desc))
             continue;
         for (int i = 0; i < attrsNum; i++) {
-            auto &attr = attrs[i];
+            auto &attr = *pInitAttrs[i];
```

Comment:
Couple of generic comments:
1. addZeroPoints can be extracted from the loop and called once for pAttr
2. We don't need to apply both Attr types for each desc. Binary Attrs should be used for brgemm kernels and lecacy Attrs for legacy kernels. That should improve first inference latency metric.

I am ok to merge the PR as is, but lets consider this improvements in future.

Reply:
Got it.